gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -361,4 +355,20 @@ trait GlutenTestsTrait extends GlutenTestsCommonTrait {
       _spark.sparkContext.parallelize(Seq(inputRow)),
       StructType(structFileSeq.toSeq))
   }
+
+  protected def checkEvaluationWithFallback(
+      expression: => Expression,
+      expected: Any,
+      inputRow: InternalRow = EmptyRow): Unit = {
+    val resolver = ResolveTimeZone
+    val expr = resolver.resolveTimeZones(expression)
+    val catalystValue = CatalystTypeConverters.convertToCatalyst(expected)
+    checkEvaluationWithoutCodegen(expr, catalystValue, inputRow)
```

Comment:
It seems these function are testing vaniila Cast, not related to gluten. Why we need it here?

Reply:
I'm trying to reproduce the problem with `checkExecution`, but the local compilation environment has some problems. The tests added to gluten should be related to testing gluten trasformer. If a case is just for testing vanilla, there is really no need to add it.
cc @PHILO-HE

Reply:
For now, It seems useless. We may consider adding it later once we have a better reason.